internal/engine/eval/templates/regoDenyByDefaultTemplate.tmpl

```diff
@@ -0,0 +1 @@
+Evaluation {{ .status }}: {{ .message }}{{ with .entityName }} for entity {{ . }}{{ end }}
```

Comment:
Based on https://github.com/stacklok/minder-stories/issues/42#issuecomment-2394190172 should this be `Evaluation {{ .status }}: {{ .message }}{{ with .entityName }} for {{ . }}{{ end }}`?

Reply:
Also, should we remove the `Evaluation {{ .status }}:` prefix? I think all the places that would use this already show the status separately from the details.

Reply:
Yes, I might have prematurely optimized for the other cases (non `failed` ones).
I'll remove that.